_data/trishitapingolia/trishitapingolia.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Trishita Pingolia
+institution: Dr.Akhilesh Das Gupta Institute Of Technology & Management
```

Comment:
Please follow the given [instructions](https://github.com/education/graduation#3-add-your-profile-information) by removing the special character `&` from this line.

Reply:
Special characters removed.